docs/examples.md

```diff
@@ -41,6 +41,51 @@ pipx run --spec https://github.com/psf/black/archive/18.9b0.zip black --help
 pipx run https://gist.githubusercontent.com/cs01/fa721a17a326e551ede048c5088f9e0f/raw/6bdfbb6e9c1132b1c38fdd2f195d4a24c540c324/pipx-demo.py
 ```
 
+You can run local files, or scripts hosted on the internet, and you can run them with arguments:
+
+```
+pipx run test.py
+pipx run test.py 1 2 3
+pipx run https://example.com/test.py
+pipx run https://example.com/test.py 1 2 3
```

Comment:
I’m not sure we want to mention this in documentation since running arbitrary code directly from the internet is very dangerous to attacks such as MITM. This is similar to e.g. what we used to recommend (`curl .../get-pip.py | python`) but now split the command into two. It is OK for this to be supported (use it at your own risk if you managed to find it in `--help`) but I would like to see this removed from documentation. Or at least we should move it to a separate section and mention very prominently this is dangerous.

Also, while I’m writing this, I wonder if `pipx run - [args]` is a good thing to support.

Reply:
We can already run arbitrary code, see [here](https://github.com/pypa/pipx#running-from-url). All this feature adds is the ability to define requirements.

> Also, while I’m writing this, I wonder if `pipx run - [args]` is a good thing to support.

I don't have a strong opinion on this, but I do think it can be handled in a follow-up PR if you think it's worth doing.

Reply:
@uranusjr is there anything you want me to do to address this? I could just remove the two examples with URLs, but as I said this isn't a new feature, and it's already documented in the last line of the block of examples jutst above what I added.